raster/src/main/scala/geotrellis/raster/io/arg/ArgReader.scala

```diff
@@ -113,72 +101,73 @@ object ArgReader {
     }
   }
 
-  final def read(path: String, typ: CellType, cols: Int, rows: Int): Tile = {
-    ArrayTile.fromBytes(Filesystem.slurp(path), typ, cols, rows)
+  final def read(path: String, cellType: CellType, cols: Int, rows: Int): Tile = {
+    ArrayTile.fromBytes(Filesystem.slurp(path), cellType, cols, rows)
   }
 
-  final def read(path: String, typ: CellType, rasterExtent: RasterExtent, targetExtent: RasterExtent): Tile = {
-    val size = typ.numBytes(rasterExtent.size)
+  final def read(path: String, cellType: CellType, rasterExtent: RasterExtent, targetExtent: RasterExtent): Tile = {
```

Comment:
This is perhaps tough to do, given the constraints that supporting a BitType force upon us